### ISSUE TITLE
Runnig with python3

### ISSUE THREAD

**Issue by paffell** [MichalBusta/DeepTextSpotter#6]
Hi,

is there a way to run the code with python3.5. 

Thanks

**Comment by MichalBusta** [comment]
Hi, 
you can just compile caffe with python3 (but - I gave it first try on ubuntu and there are linkage problems - somehow python 2.x gets involved = some fixing of cmake files will be needed). 

Michal

**Comment by paffell** [comment]
Not sure wheter you had the same problem as me. I was able to successfully build it, but the problem is that Py_InitModule is no longer used in python3. However, I am not an expert in writing cpp extensions, but I will give it a try

**Comment by MichalBusta** [comment]
Not same problem: I had problem with building caffe itself - the extension module is simple - and for most application you don't need it - it is just prefix decoding with dictionary - so you can just comment out the sections with cmp_trie

**Comment by paffell** [comment]
Thank you for the info, I will try, but it seems I solved it just by defining PYTHON3 in trie_api.h. However, I have got another error: `Error parsing text-format caffe.NetParameter: 513:14: Message type "caffe.LayerParameter" has no field named "reorg_param".`

**Comment by MichalBusta** [comment]
Have you switch to darknet branch of caffe?

**Comment by paffell** [comment]
No, I am using standard caffe's master branch. How can I do that?

**Comment by MichalBusta** [comment]
Readme.md -> Installation step 1 - merge to branch darknet to your caffe or build provided (https://github.com/MichalBusta/caffe - branch darknet) .

**Comment by paffell** [comment]
Got it, as I already had caffe installation, I skip this step and missed it. I am currently having a trouble to build darknet caffe, but hopefully I will solve it.

**Comment by parquets** [comment]
@paffell  so, do you save the problem? I meet the same problem and i do as the README. However, I meet some problem while make. My Terminal shows as " undefine reference...TIFF... " while make at 84%.
